modules/mobile/spec/request/appointments_cancel_request_spec.rb

```diff
@@ -32,12 +32,44 @@
     end
 
     context 'when feature flag is on and user has access' do
-      it 'returns no content' do
-        VCR.use_cassette('mobile/appointments/VAOS_v2/cancel_appointment_vpg_200', match_requests_on: %i[method uri]) do
-          VCR.use_cassette('mobile/appointments/VAOS_v2/get_facilities_200', match_requests_on: %i[method uri]) do
-            put "/mobile/v0/appointments/cancel/#{cancel_id}", params: nil, headers: sis_headers
+      context 'using VAOS' do
+        before do
+          Flipper.disable(:va_online_scheduling_enable_OH_cancellations)
+          Flipper.disable(:va_online_scheduling_use_vpg)
+        end
 
-            expect(response).to have_http_status(:no_content)
+        it 'returns no content' do
+          VCR.use_cassette('mobile/appointments/VAOS_v2/cancel_appointment_200', match_requests_on: %i[method uri]) do
+            VCR.use_cassette('mobile/appointments/VAOS_v2/get_facilities_200', match_requests_on: %i[method uri]) do
+              put "/mobile/v0/appointments/cancel/#{cancel_id}", params: nil, headers: sis_headers
+
+              puts response.body
```

Comment:
Looks like a couple debugging `puts` got left in. otherwise looks good for mobile!